sdk/src/main/java/com/swedbankpay/mobilesdk/internal/WebViewFragment.kt

```diff
@@ -193,13 +281,41 @@ internal class WebViewFragment : Fragment() {
         }
     }
 
-    class ErrorDialogFragment : DialogFragment() {
+    class IntentUriException(
+        val uri: Uri,
+        message: String?,
+        cause: Throwable?
+    ) : Exception(message, cause)
+
+    class IntentUriErrorDialogFragment : DialogFragment() {
         companion object {
             const val ARG_URI = "U"
+            const val ARG_EXTRA_MESSAGE = "E"
 
-            fun newInstance(uri: Uri) = ErrorDialogFragment().apply {
+            fun newInstance(exception: IntentUriException, verbose: Boolean) = IntentUriErrorDialogFragment().apply {
```

Comment:
Codacy found an issue: [UndocumentedPublicFunction - [newInstance] at /src/sdk/src/main/java/com/swedbankpay/mobilesdk/internal/WebViewFragment.kt:295:17](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-android/pullRequest?prid=6032614)

Reply:
Codacy apparently does not understand internal functions (or classes, etc.) properly. This is not a public function.